fhir-server/src/test/java/au/csiro/pathling/fhirpath/function/WhereFunctionTest.java

```diff
@@ -114,27 +124,33 @@ public void whereOnElement() {
         .withRow("Patient/abc3", "en")
         .withRow("Patient/abc3", "en")
         .withRow("Patient/abc3", "zh")
+        .withRow("Patient/abc4", "fr")
+        .withRow("Patient/abc4", "fr")
```

Comment:
There is no row with NULL value.

Reply:
See 9fb1a6e29c01238ea501bff04ba6a2cdd97bc3f2.